docs/guides/js-waku/light-send-receive.md

```diff
@@ -135,6 +159,16 @@ const subscription = await node.filter.createSubscription();
 await subscription.subscribe([decoder], callback);
 ```
 
+The `pubsubTopicShardInfo` parameter allows you to configure a different shared pubsub topic for your `Filter` subscription:
+
+```js
+// Create the shard info
+const shardInfo = { clusterId: 3, shard: 1 };
+
+// Create Filter subscription with custom shard info
+const subscription = await node.filter.createSubscription(shardInfo);
```

Comment:
just a heads up that it will change soon as the current API seems to be too hard to understand. Maybe we can delay this PR or we can follow up on it. I think we will have it updated in a week or so